src/collection-facets/facet-row.ts

```diff
@@ -0,0 +1,271 @@
+import {
+  css,
+  html,
+  LitElement,
+  TemplateResult,
+  CSSResultGroup,
+  nothing,
+} from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import type { CollectionNameCacheInterface } from '@internetarchive/collection-name-cache';
+import eyeIcon from '../assets/img/icons/eye';
+import eyeClosedIcon from '../assets/img/icons/eye-closed';
+import type {
+  FacetOption,
+  FacetBucket,
+  FacetEventDetails,
+  FacetState,
+} from '../models';
+
+@customElement('facet-row')
+export class FacetRow extends LitElement {
```

Comment:
New `<facet-row>` component handles rendering & interactions for a single facet bucket. Code is mostly just migrated from `facets-template`, with a bit of cleanup.